src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -439,8 +538,23 @@ void OwncloudAdvancedSetupPage::slotStyleChanged()
 
 void OwncloudAdvancedSetupPage::customizeStyle()
 {
-    if(_progressIndi)
-        _progressIndi->setColor(QGuiApplication::palette().color(QPalette::Text));
+    if (_progressIndi) {
```

Comment:
I'd argue `_progressIndi` is a bad member name, I have no idea what this means.

Reply:
yes, I guess it means progress indicator;) but I thought I change as less as possible.

Reply:
Oh right, it was already there, you didn't introduce it. Nevermind then.